circletree/CircleSegment.jsx

```diff
@@ -75,9 +82,29 @@ var CircleSegment = React.createClass({
     }
   },
 
+  onActivate(e) {
+    this.setState({
+      status: ""
+    });
+    if (e.detail.origin === this) {
+      this.activate("primary");
```

Comment:
`"primary active"`? without the "active" part, CSS filtering for `:not(.active)` becomes a lot harder